composer.lock

```diff
@@ -954,26 +954,26 @@
         },
         {
             "name": "composer/pcre",
-            "version": "2.2.0",
+            "version": "2.3.1",
```

Comment:
These changes in lock file are unrelated to this PR.

Reply:
yes, I've missed and used composer update.